src/github/types/webhook-events.ts

```diff
@@ -1,2 +1,23 @@
-import { emitterEventNames } from "@octokit/webhooks/dist-types/generated/webhook-names.js";
+import { emitterEventNames } from "@octokit/webhooks";
 export type GitHubEventClassName = (typeof emitterEventNames)[number];
+
+type Formatted<T extends string> = T extends `${infer Prefix}.${infer Rest}` ? `${Prefix}_${Formatted<Rest>}` : T;
+
+type GithubEventWebHookEvents = {
+  [K in Formatted<Uppercase<GitHubEventClassName>>]: Formatted<Lowercase<K>>;
```

Comment:
```suggestion
  [K in GitHubEventClassName as Formatted<Uppercase<K>>]: K;
```
leave value original and just format the key